### PR TITLE
Added Static and Css folders to the resources folder

### PR DESCRIPTION
**Pull request by MachariaKuria**

### CODE REVIEW

File: src/main/resources/static/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  display: flex;
+  float: left;
+  width: 50%;
+  padding: 12px;
+}
+div{
+  position:relative;
+  min-height: 280px;
+  width: 100%;
+  background-color: background-color: #A9C9FF;
```
[the remark this answers is not in the document]
Comment (IzzyElwyn) [reply src/main/resources/static/css/styles.css]: @MachariaKuria looks like you've accidentally repeated background-color here

Comment (IzzyElwyn) [inline src/main/resources/static/css/styles.css]: Hey Simon! 

I think the only other thing to fix on this is to remove the repeated background-color: on line 57. then we should be in good shape to merge :)

Comment (IzzyElwyn) [reply src/main/resources/static/css/styles.css]: You'll need to do a 'git push' so it disappears from the pr

Comment (MachariaKuria) [reply src/main/resources/static/css/styles.css]: I did a git push. I got an error message saying everything up-to-date

**Pull request merged**